### PR TITLE
[n/a] add configuration file for redis to up the limit of database

### PR DESCRIPTION
**Pull request by ker0x**
By default, Redis limits the number of databases to 16. I have increased this limit to 100 so that I can apply index ranges from 20 to 20 per project/brand

### CODE REVIEW

**Review by maxhelias — APPROVED** [review]

File: redis/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM redis:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
+
+CMD [ "redis-server", "/usr/local/etc/redis/redis.conf" ]
+
+RUN usermod -u 1000 www-data
```
Comment (ajardin) [inline redis/Dockerfile]: I don't think it's needed for a redis container. Also, could you add a newline at the end of file?

File: redis/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM redis:latest
+
```
Comment (ajardin) [inline redis/Dockerfile:2]: Could you add something about the maintainer? Maybe a generic address related to the team. 😉 

`LABEL maintainer="XXXXX XXXXX <xxxxx@emakina.fr>"`

File: redis/redis.conf [hunk 3]
```diff
@@ -0,0 +1 @@
+databases 100
```
Comment (ajardin) [inline redis/redis.conf]: Could you add a newline at the end of file?

**Review by ajardin — CHANGES_REQUESTED** [review]

File: redis/Dockerfile (hunk 2 above)
Comment (ker0x) [reply redis/Dockerfile:2]: @ajardin fixed

File: redis/redis.conf (hunk 3 above)
Comment (ker0x) [reply redis/redis.conf]: @ajardin fixed

**Review by ajardin — APPROVED** [review]

**Review by maxhelias — CHANGES_REQUESTED** [review]
"MAINTAINER" is obsolete, just use "LABEL"

**Comment by ker0x** [comment]
@maxhelias fixed for all Dockerfile

**Review by maxhelias — APPROVED** [review]

**Pull request merged**